distribution/test/itests/README.md

```diff
@@ -8,14 +8,14 @@ mvn clean verify -DisDebugEnabled=true
 ```
 
 ## SSH Into a Running Instance
-It is possible to SSH into a running test instance.  This will allow you to use the shell to inspect the runtime state while the test probe is installed and running.
+It is possible to SSH into a running test instance. This will allow you to use the shell to inspect the runtime state while the test probe is installed and running. The SSH port is dynamic and can be found in `target/exam/<GUID>/etc/org.apache.karaf.shell.cfg`.
 
 ```
-ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -p 9101 admin@localhost
+ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -p 20003 admin@localhost
 ```
 
 ## Debug a Single Test
-The Pax Exam tests support Maven Surefire Plugin properties.  One useful property is the `test` property to select a single test class or method to execute.
+The Pax Exam tests support Maven Surefire Plugin properties. One useful property is the `test` property to select a single test class or method to execute.
```

Comment:
Looks like I forgot to change `test` to `it.test` here when I updated this last time.